### PR TITLE
update weights

### PR DESCRIPTION
**Pull request by coderobe**

### CODE REVIEW

File: runtime/polkadot/src/weights/frame_system.rs [hunk 1]
```diff
@@ -57,29 +59,32 @@ impl<T: frame_system::Config> frame_system::WeightInfo for WeightInfo<T> {
 	// Storage: System Digest (r:1 w:1)
 	// Storage: unknown [0x3a686561707061676573] (r:0 w:1)
 	fn set_heap_pages() -> Weight {
-		(2_347_000 as Weight)
+		(5_134_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
```
Comment (chevdor) [inline runtime/polkadot/src/weights/frame_system.rs:64]: x2

File: runtime/polkadot/src/weights/pallet_democracy.rs [hunk 2]
```diff
@@ -94,44 +97,47 @@ impl<T: frame_system::Config> pallet_democracy::WeightInfo for WeightInfo<T> {
...
 	// Storage: Democracy NextExternal (r:0 w:1)
 	fn external_propose_default() -> Weight {
-		(1_254_000 as Weight)
+		(3_982_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	// Storage: Democracy NextExternal (r:1 w:1)
 	// Storage: Democracy ReferendumCount (r:1 w:1)
 	// Storage: Democracy ReferendumInfoOf (r:0 w:1)
 	fn fast_track() -> Weight {
```
Comment (chevdor) [inline runtime/polkadot/src/weights/pallet_democracy.rs:131]: noticeable

File: runtime/polkadot/src/weights/pallet_scheduler.rs [hunk 3]
```diff
@@ -48,10 +48,11 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
...
 	fn on_initialize_periodic_named_resolved(s: u32, ) -> Weight {
-		(675_000 as Weight)
-			// Standard Error: 31_000
-			.saturating_add((24_318_000 as Weight).saturating_mul(s as Weight))
+		(2_739_000 as Weight)
+			// Standard Error: 32_000
+			.saturating_add((26_230_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().reads((3 as Weight).saturating_mul(s as Weight)))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
```
Comment (chevdor) [inline runtime/polkadot/src/weights/pallet_scheduler.rs:58]: noticeable

File: runtime/polkadot/src/weights/pallet_scheduler.rs [hunk 4]
```diff
@@ -73,10 +75,11 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
...
 	fn on_initialize_periodic_resolved(s: u32, ) -> Weight {
-		(0 as Weight)
-			// Standard Error: 30_000
-			.saturating_add((22_043_000 as Weight).saturating_mul(s as Weight))
+		(1_851_000 as Weight)
+			// Standard Error: 29_000
+			.saturating_add((23_826_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().reads((3 as Weight).saturating_mul(s as Weight)))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
```
Comment (chevdor) [inline runtime/polkadot/src/weights/pallet_scheduler.rs:85]: 0 to 1.8m

File: runtime/polkadot/src/weights/pallet_scheduler.rs [hunk 5]
```diff
@@ -85,10 +88,11 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
...
+	/// The range of component `s` is `[1, 50]`.
 	fn on_initialize_resolved(s: u32, ) -> Weight {
-		(248_000 as Weight)
-			// Standard Error: 28_000
-			.saturating_add((18_646_000 as Weight).saturating_mul(s as Weight))
+		(4_340_000 as Weight)
+			// Standard Error: 33_000
+			.saturating_add((20_124_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().reads((2 as Weight).saturating_mul(s as Weight)))
```
Comment (chevdor) [inline runtime/polkadot/src/weights/pallet_scheduler.rs:97]: noticeable

File: runtime/polkadot/src/weights/runtime_parachains_disputes.rs [hunk 6]
```diff
@@ -46,7 +46,7 @@ pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> runtime_parachains::disputes::WeightInfo for WeightInfo<T> {
 	// Storage: ParasDisputes Frozen (r:0 w:1)
 	fn force_unfreeze() -> Weight {
-		(659_000 as Weight)
+		(3_428_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 }
```
Comment (chevdor) [inline runtime/polkadot/src/weights/runtime_parachains_disputes.rs:52]: noticeable

File: runtime/polkadot/src/weights/runtime_parachains_paras.rs [hunk 7]
```diff
@@ -58,8 +59,9 @@ impl<T: frame_system::Config> runtime_parachains::paras::WeightInfo for WeightIn
 			.saturating_add(T::DbWeight::get().writes(6 as Weight))
 	}
 	// Storage: Paras Heads (r:0 w:1)
+	/// The range of component `s` is `[1, 1048576]`.
 	fn force_set_current_head(s: u32, ) -> Weight {
-		(9_108_000 as Weight)
+		(0 as Weight)
 			// Standard Error: 0
 			.saturating_add((1_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
```
Comment (chevdor) [inline runtime/polkadot/src/weights/runtime_parachains_paras.rs:67]: 9m -> 0

File: runtime/polkadot/src/weights/runtime_parachains_paras.rs [hunk 8]
```diff
@@ -85,8 +88,9 @@ impl<T: frame_system::Config> runtime_parachains::paras::WeightInfo for WeightIn
 	// Storage: Paras FutureCodeUpgrades (r:1 w:0)
 	// Storage: Paras Heads (r:0 w:1)
 	// Storage: Paras UpgradeGoAheadSignal (r:0 w:1)
+	/// The range of component `s` is `[1, 1048576]`.
 	fn force_note_new_head(s: u32, ) -> Weight {
-		(12_655_000 as Weight)
+		(0 as Weight)
 			// Standard Error: 0
 			.saturating_add((1_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
```
Comment (chevdor) [inline runtime/polkadot/src/weights/runtime_parachains_paras.rs:96]: 12m -> 0

**Review by chevdor — APPROVED** [review]

File: runtime/polkadot/src/weights/pallet_scheduler.rs (hunk 3 above)
Comment (coderobe) [reply runtime/polkadot/src/weights/pallet_scheduler.rs:58]: this base weight is insignificant

File: runtime/polkadot/src/weights/pallet_scheduler.rs (hunk 4 above)
Comment (coderobe) [reply runtime/polkadot/src/weights/pallet_scheduler.rs:85]: this base weight is insignificant

**Pull request merged**